packages/components/.storybook/preview-body.html

```diff
@@ -0,0 +1,9 @@
+<style>
```

Comment:
follows https://storybook.js.org/docs/react/configure/story-rendering#adding-to-body